### PR TITLE
Add feature `remove-diff-signs`

### LINKED ISSUE #1464: Add opt-in setting to remove diff signs
There is (or [was](https://github.com/sindresorhus/refined-github/pull/1451)) a feature called [`remove-diff-signs.js`](https://github.com/sindresorhus/refined-github/blob/master/source/features/remove-diff-signs.js), but it's [intention](https://github.com/sindresorhus/refined-github/pull/1451#issuecomment-409374499) is to make the diff signs unselectable. Personally I think those signs are redundant in combination with the background colors in the diffs and can be removed.

If this feature request is approved, then the following code is probably all you need to include:
```css
.blob-code-marker-addition:before,
.blob-code-marker-context:before,
.blob-code-marker-deletion:before {
	display: none !important;
}
```

### PR DESCRIPTION
**Pull request by bfred-it**
Closes #1464 

It can be disabled 🎈 

# Test

Any commits on https://github.com/sindresorhus/refined-github/commits/master
Any reviewed block on https://github.com/sindresorhus/refined-github/pull/1783
Full diff on https://github.com/sindresorhus/refined-github/pull/1783/files

```diff
  inline diffs
- minus
+ plus
```
```patch
diff --git a/foo.c b/foo.c
index 30cfd169..8de130c2 100644
--- a/foo.c
+++ b/foo.c
@@ -1,5 +1,5 @@
 #include <string.h>

 int check (char *string) {
-    return !strcmp(string, "ok");
+    return (string != NULL) && !strcmp(string, "ok");
 }
```

# Question

Where else do commits appear

# Screenshot

screenshot

### CODE REVIEW

File: source/features/extend-diff-expander.tsx [hunk 1]
```diff
@@ -16,7 +16,8 @@ function init() {
 features.add({
 	id: 'extend-diff-expander',
 	include: [
-		features.isPRFiles
+		features.isPRFiles,
+		features.isCommit
```
Comment (bfred-it) [inline source/features/extend-diff-expander.tsx]: Unrelated change, but I saw that this was supposed to also work on commit pages

**Comment by bfred-it** [comment]
This is ready to merge, except I need suggestions on how to handle diffs and patches. This is how it looks currently.

Screenshot 2019-03-22 at 16 04 25

Two notes:

- <code>\`\`\`diff</code> and <code>\`\`\`patch</code> are indistinguishable from the dom
- `patch` will show those `---`/`+++` at the start and they're not removable

**Comment by sindresorhus** [comment]
> Perhaps we should leave inline diffs/patches untouched.

Yeah, probably best. I don't think people would realize they could still copy-paste it and use it as a `.diff` with Git if they don't see the `+` and `-`.

**Comment by sindresorhus** [comment]
The downside of hiding the diff symbols is that it now looks like all code is indented. Should we also move it to the left?

**Comment by sindresorhus** [comment]
> Where else do commits appear

Change suggestions: https://github.com/sindresorhus/package-json/pull/51#discussion_r268386873 (Works)

**Comment by bfred-it** [comment]
> The downside of hiding the diff symbols is that it now looks like all code is indented. Should we also move it to the left?

I'm hesitant to do that. Not all code blocks are indented the same way (some use `padding-left`, some plain text nodes) and other extensions might expect the space to be there:



Remember what we had to go through with the previous `remove-diff-signs`

**Comment by sindresorhus** [comment]
> Remember what we had to go through with the previous remove-diff-signs

Yeah... But that was slightly different from what I can remember. Then, we actually replaced some nodes. This would only be CSS changes.

**Comment by nesl247** [comment]
Commenting here since I was about to open a bug about it. The new spacing is much worse IMO than having the symbols. The spacing causes me to believe everything is formatted badly when reading the diffs and I have to remember that the space is there because of this feature.

**Pull request merged**